src/planwise/endpoint/projects2.clj

```diff
@@ -64,6 +65,7 @@
        (if (nil? project)
          (not-found {:error "Project not found"})
          (do
+           (assert (s/valid? ::project-start-model/project-starting project) "Invalid starting project")
```

Comment:
This will throw and generate a 500, but this should actually be a 400 since it's a bad request from the client.